src/material-experimental/mdc-menu/_menu-theme.scss

```diff
@@ -53,7 +53,7 @@
     .mat-mdc-menu-content {
       // Note that we include this private mixin, because the public
       // one adds a bunch of styles that we aren't using for the menu.
-      @include mdc-list.deprecated-base_(mdc-helpers.$mat-typography-styles-query);
+      @include mdc-list.list-base(mdc-helpers.$mat-typography-styles-query);
```

Comment:
It looks like the typography override below broke, it needs to target `.mat-mdc-menu-item .mdc-list-item__primary-text` now